WordPress/src/main/java/org/wordpress/android/ui/prefs/accountsettings/AccountSettingsViewModel.kt

```diff
@@ -42,17 +50,21 @@ class AccountSettingsViewModel @Inject constructor(
     private val resourceProvider: ResourceProvider,
     networkUtilsWrapper: NetworkUtilsWrapper,
     @Named(UI_THREAD) private val mainDispatcher: CoroutineDispatcher,
+    @Named(BG_THREAD) private val bgDispatcher: CoroutineDispatcher,
     private val fetchAccountSettingsUseCase: FetchAccountSettingsUseCase,
     private val pushAccountSettingsUseCase: PushAccountSettingsUseCase,
     private val getAccountUseCase: GetAccountUseCase,
     private val getSitesUseCase: GetSitesUseCase,
-    private val optimisticUpdateHandler: AccountSettingsOptimisticUpdateHandler
+    private val optimisticUpdateHandler: AccountSettingsOptimisticUpdateHandler,
+    private val accountRestClient: AccountRestClient,
```

Comment:
The implementation worked as expected for me but I've noticed a compilation error on the `AccountSettingsViewModelTest` test code. Trying to come up with a quick fix for the test I realised that it might be easier to mock the call by using a Use Case (like we do for fetching/pushing account settings). Wdyt?

Reply:
Great idea! I have addressed this (https://github.com/wordpress-mobile/WordPress-Android/pull/18457/commits/93f9741b49c1d1b8572effd0057abc442a70accd), along with adding some unit tests in [this PR](https://github.com/wordpress-mobile/WordPress-Android/pull/18457).

Reply:
Thank you Matthew 🙇